src/App.js

```diff
@@ -1,13 +1,96 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      userInput: '',
+      backgroundColors: ['#f1c40f', '#e67e22', '#e74c3c', '#1abc9c', '#2ecc71', '#3498db', '#9b59b6'],
```

Comment:
Really like this idea of color coding the todos. In the future, you could build this out even more by giving each task a category and tying the colors to those categories. 👍